apps/site/pages/en/download/archive.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: Download Node.js®
+layout: download-archive
+---
+
+<WithDownloadArchive>
+  {({ binaries, installers, version, release, sources }) => (
+    <>
+      <h1>Node.js® Download Archive</h1>
+
+      <h2 className="flex items-center gap-2">
+        <img
+          src="/static/images/favicons/favicon.png"
```

Comment:
do we have an SVG ?

Reply:
this ones? https://nodejs.org/en/about/branding#js-icons

Reply:
I believe Augustin is asking if we have an svg for the Node.js icon in `@node-core/ui-components`. No, we do not have a Node.js icon-only component